docs/sandbox/MapboxVectorTilesApi.md

```diff
@@ -20,11 +20,20 @@ Currently, only the language with the highest priority from the header is used.
 
 ### Configuration
 
-To enable this you need to add the feature `SandboxAPIMapboxVectorTilesApi` in `otp-config.json`.
+To enable this you need to add the feature `otp-config.json`.
+
+```json
+// otp-config.json
+{
+  "otpFeatures": {
+    "VehicleToStopHeuristics": true
```

Comment:
```suggestion
    "SandboxAPIMapboxVectorTilesApi": true
```

Reply:
Thanks for this. It was a stupid copypaste error.